imap/append.c

```diff
@@ -1107,7 +1107,10 @@ EXPORTED int append_fromstage_full(struct appendstate *as, struct body **body,
         goto out;
     }
 
-    if (config_getstring(IMAPOPT_ANNOTATION_CALLOUT)) {
+    if (
+        config_getstring(IMAPOPT_ANNOTATION_CALLOUT)
```

Comment:
Just a nit.  We (the core devs) typically don't put like breaks after the opening '(' and before the closing ')'

Reply:
Happy to address.  (Also, eager to see what autoformatting we could agree on…)

Reply:
@ksmurchison I believe my latest commit matches the style I've now found elsewhere in append.c.  Look right to you?

Reply:
Yes sir.  All good!